pyproject.toml

```diff
@@ -12,7 +12,7 @@ keywords = ["merge request", "gitlab"]
 python = "^3.7"
 ConfigArgParse = "^1.3"
 maya = "^0.6.1"
-PyYAML = "^5.4.1"
+PyYAML = "~5.3.1"
```

Comment:
Note to self: tilde here essentially means `>= 5.3.1 <5.4.0`, i.e. allow _patch-level_ updates only but nothing else.